Revert-tvOS/Sources/HomeCollectionViewController.swift

```diff
@@ -28,33 +43,30 @@ final class HomeCollectionViewController: UICollectionViewController {
         fatalError("`SettableHomeItem` requires `indexPath` to be sent as the sender.")
       }
 
-      destinationViewController.item = self.collection[indexPath]
+      destinationViewController.item = self.dataSource[indexPath]
     }
   }
 
   // MARK: - Private
-  private let collection = CollectableCollection<HomeItem>(items: .HomeCollection)
   private let dataSource: CollectionDataSource<HomeItem, HomeCollectionCell>
 }
  
 private extension HomeCollectionViewController {
   static func configureCell(cell: HomeCollectionCell, withItem item: HomeItem) {
     cell.titleLabel.text = item.title
-    
-    // TODO: remove **the check** later on. This is just for testing purposes while we don't have images ready.
-    if let iconImage = UIImage(named: item.iconName) {
-      cell.imageView.image = iconImage
-    } else {
-      cell.imageView.image = UIImage(named: "orange")
-    }
+    cell.imageView.image = UIImage(named: item.iconName)
   }
 }
 
 // MARK:- UICollectionViewDelegate
 extension HomeCollectionViewController {
   override func collectionView(collectionView: UICollectionView, didSelectItemAtIndexPath indexPath: NSIndexPath) {
-    let item = self.collection[indexPath]
+    let item = self.dataSource[indexPath]
 
     self.performSegueWithIdentifier(item.segueIdentifier, sender: indexPath)
   }
 }
+
+protocol SettableHomeCollectionGroup: class {
```

Comment:
What about `GroupFilterable` ? Exposing a `groupFilter` `String`.

You probably did this because of my poorly named `SettableHomeItem`. I'm open to suggestions to make that better too :P